dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/ProgramRuleActionObjectBundleHook.java

```diff
@@ -69,6 +70,21 @@ public void validate(
     }
   }
 
+  @Override
+  public void preCreate(ProgramRuleAction object, ObjectBundle bundle) {
```

Comment:
Why are we getting ProgramNotificationTemplate from the DB?
Should be already in ProgramRuleAction at this point?

Reply:
No. It will be `ProgramNotificationTemplate` object with only uid.